File: doc/codes/python/examples/api/command/output-mask.py
from autd3.commands import SetOutputMask
from autd3.geometry import Autd3, Geometry

geometry = Geometry([Autd3([0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0])])

masks = [[True] * geometry.device(i).num_transducers() for i in range(geometry.num_devices())]

# ANCHOR: api
SetOutputMask(masks)
# ANCHOR_END: api

File: doc/codes/rust/examples/api/command/output-mask.rs
use autd3_rs::commands::SetOutputMask;
use autd3_rs::geometry::{Autd3, Geometry};

fn main() {
    let geometry = Geometry::new(vec![Autd3::default()]);

    let masks: Vec<Vec<bool>> = geometry
        .iter()
        .map(|dev| vec![true; dev.num_transducers()])
        .collect();

    // ANCHOR: api
    SetOutputMask { masks: &masks };
    // ANCHOR_END: api
}
